genesyscloud/webdeployments_configuration/resource_genesyscloud_webdeployments_configuration_schema.go

```diff
@@ -356,6 +356,26 @@ var (
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
+			"pause_criteria": {
+				Description: "List of pause criteria",
```

Comment:
The public API description is more useful here I think - "Pause criteria that will pause cobrowse if some of them are met in the user's URL". The docs will then indicate that it is a list

Reply:
done

Reply:
I didnt know about go generate! great thing : D I did it manually